scripts/pi-hole/js/formatter.mjs

```diff
@@ -0,0 +1,18 @@
+/* Pi-hole: A black hole for Internet advertisements
+ *  (c) 2024 Pi-hole, LLC (https://pi-hole.net)
+ *  Network-wide ad blocking via your own hardware.
+ *
+ *  This file is copyright under the latest version of the EUPL.
+ *  Please see LICENSE file for your rights under this license. */
+
+export function toPercent(number, fractionDigits = 0) {
+    const userLocale = navigator.language || "en-US";
+    return new Intl.NumberFormat(userLocale, {
+        style: "percent",
+        minimumFractionDigits: fractionDigits,
+        maximumFractionDigits: fractionDigits,
+    }).format(number / 100);
+}
+
+// Appease the CI
+toPercent(0);
```

Comment:
What is this doing? We shouldn't be doing things like this just to appease the CI. What part of the CI failed without this?

Have just tested locally and it is still giving the same error as current with or without this line.

Reply:
It is complaining that the function is set but unused.